### PR TITLE
Added npc functions

### PR DESCRIPTION
**Pull request by StrawWagen**
e:npcGetCurrentSchedule() -Gets npc's schedule
E:npcSetSchedule(n) -Sets npc's schedule

### CODE REVIEW

File: lua/entities/gmod_wire_expression2/core/custom/cl_cfc_e2_lib.lua [hunk 1]
```diff
@@ -65,7 +65,9 @@ E2Helper.Descriptions["npcSetDamageMultiplier(e:n)"]   = "Sets the damage multip
...
 E2Helper.Descriptions["npcCreate(sv)"]                 = "Creates an npc of class S, at position V. Admin only."
 E2Helper.Descriptions["npcCreate(svs)"]                = "Creates an npc of class S1, at position V, setting S2 to the key value 'model'. Admin only."
+E2Helper.Descriptions["npcSetSchedule(e:n)"]           = "Sets an npc's schedule, See Enums/SCHED. Admin only."
 E2Helper.Descriptions["npcGetGlobalSquad(e:)"]         = "Returns an NPC's squad."
 E2Helper.Descriptions["npcGetWeaponProficiency(e:)"]   = "Returns the weapon proficiency of the NPC."
-E2Helper.Descriptions["npcGetDamageMultiplier(e:)"]    = "Returns the damage multiplier of the NPC"
-E2Helper.Descriptions["npcCouldNotReach(e:e)"]         = "Returns 1 if the NPC marked ent2 as unreachable"
+E2Helper.Descriptions["npcGetDamageMultiplier(e:)"]    = "Returns the damage multiplier of the NPC."
+E2Helper.Descriptions["npcCouldNotReach(e:e)"]         = "Returns 1 if the NPC marked ent2 as unreachable."
+E2Helper.Descriptions["npcGetCurrentSchedule(e:)"]     = "Returns the npc's current schedule. See Enums/SCHED"
```
Comment (StrawWagen) [inline lua/entities/gmod_wire_expression2/core/custom/cl_cfc_e2_lib.lua:73]: e2helper for getCurrentSchedule()

File: lua/entities/gmod_wire_expression2/core/custom/cl_cfc_e2_lib.lua [hunk 2]
```diff
@@ -65,7 +65,9 @@ E2Helper.Descriptions["npcSetDamageMultiplier(e:n)"]   = "Sets the damage multip
 E2Helper.Descriptions["npcSetGlobalSquad(e:s)"]        = "Sets the squad of an NPC. Admin only."
 E2Helper.Descriptions["npcCreate(sv)"]                 = "Creates an npc of class S, at position V. Admin only."
 E2Helper.Descriptions["npcCreate(svs)"]                = "Creates an npc of class S1, at position V, setting S2 to the key value 'model'. Admin only."
+E2Helper.Descriptions["npcSetSchedule(e:n)"]           = "Sets an npc's schedule, See Enums/SCHED. Admin only."
```
Comment (StrawWagen) [inline lua/entities/gmod_wire_expression2/core/custom/cl_cfc_e2_lib.lua:68]: e2helper for setSchedule

File: lua/entities/gmod_wire_expression2/core/custom/cfc_npc.lua [hunk 3]
```diff
@@ -47,6 +47,14 @@ e2function string entity:npcGetGlobalSquad()
     return squad
 end
 
+e2function number entity:npcGetCurrentSchedule()
+    if not isValidNpc( this ) then return 0 end
+    
+    local out = this:GetCurrentSchedule()
+    
+    return out
```
Comment (plally) [inline lua/entities/gmod_wire_expression2/core/custom/cfc_npc.lua:55]: A local variable like this is unnecessary especially if it just has a generic name like `out`
you can just do `return this:GetCurrentSchedule()`

**Review by plally — CHANGES_REQUESTED** [review]

File: lua/entities/gmod_wire_expression2/core/custom/cfc_npc.lua (hunk 3 above)
Comment (StrawWagen) [reply lua/entities/gmod_wire_expression2/core/custom/cfc_npc.lua:55]: oh sure

**Pull request merged**